### PR TITLE
feat: Use travis to generate gh-pages

### LINKED ISSUE #2252: Website not generated automatically - consider using Travis
Yaydoc is currently not running and it would be good to switch to Travis instead in order to generate the website automatically from the Readme.md file. Images are currently neither always correctly displayed. Please help to get the site up and running and updated automatically.

### LINKED ISSUE #2832: Fix Readme.md and gh-pages, include all assets in /docs folder
* The assets used in the main documentation need to be included directly in the /docs folder. Images should go to /docs/images.
* Please update any outdated info in the Readme as well and check that the generated site in the gh-pages branch uses the Readme.md to build the page.
* Add social media links and fix remaining links in the template used for the page.
* Get rid of left side bar and move search box to top menu bar
Screenshot from 2019-07-19 14-13-42
* Make all assets load from the same website, e.g. copy http://fossasia.org/img/fossasia-dark.png to the local repo and load it from here.

### PR DESCRIPTION
**Pull request by yashk2000**
Fixed #2832 
Fixed #2252 

Changes: Used Sphinx to make travis generate a website automatically.

### CODE REVIEW

**Comment by yashk2000** [comment]
@adityastic please review.

**Comment by adityastic** [comment]
> @adityastic please review.

on it

File: scripts/upload-gh-pages.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+git config --global user.name "Travis CI"
+git config --global user.email "noreply+travis@fossasia.org"
+
+sudo pip install Sphinx
+
+cd docs/sources
+
+make html
+
+git clone --quiet --branch=gh-pages https://fossasia:$GITHUB_KEY@github.com/fossasia/phimpme-android gh-pages > /dev/null
```
Comment (adityastic) [inline scripts/upload-gh-pages.sh]:
```suggestion
git clone --quiet --branch=gh-pages https://fossasia:$GITHUB_API_KEY@github.com/fossasia/phimpme-android gh-pages > /dev/null
```

File: scripts/upload-gh-pages.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+cd gh-pages
+
+rm -rf *
```
Comment (adityastic) [inline scripts/upload-gh-pages.sh]: empty the hidden files on gh-pages excluding .git. because rm -rf wont clear those

Comment (yashk2000) [reply scripts/upload-gh-pages.sh]: Should i use rm -rf .*

**Comment by yashk2000** [comment]
@adityastic https://github.com/fossasia/phimpme-android/pull/2854

File: scripts/upload-gh-pages.sh (hunk 2 above)
Comment (yashk2000) [reply scripts/upload-gh-pages.sh]: https://github.com/fossasia/phimpme-android/pull/2854

**Comment by yashk2000** [comment]
@iamareebjamal please review. 
I am not able to figure out why travis isn't working.

**Comment by iamareebjamal** [comment]
push a commit

**Comment by yashk2000** [comment]
@iamareebjamal I pushed a commit, but the build still didn't start. It started and passes for the new pr I sent, but not for this. What should I do?

**Comment by iamareebjamal** [comment]
I'd say reopen th

**Comment by iamareebjamal** [comment]
Recreate this PR

**Comment by yashk2000** [comment]
Ok

**Pull request closed without merging**